components/DetailedHome.jsx

```diff
@@ -53,6 +54,25 @@ export default function DetailedHome({ state, district, type }) {
 
     const [resourceChoosen, setResourceChoosen] = useState(type || 'All');
 
+    const locationResources = {
+        get All() {
+            return []
+                .concat(this.Oxygen)
+                .concat(this.Medicine)
+                .concat(this.Hospital)
+                .concat(this.Ambulance)
+                .concat(this.Helpline)
+                .concat(this.Vaccine)
+                .sort((record) => (isVerified(record.verificationStatus) ? -1 : 1));
```

Comment:
```suggestion
                .sort((record) => (isVerified(record.verification_status) ? -1 : 1));
```

From v2 API, we use `verification_status` 📍 Refer: https://github.com/coronasafe/life/wiki/Life-Data-Structure

Reply:
Renamed it to verification_status